terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf

```diff
@@ -0,0 +1,1261 @@
+  default = "galaxy-noise@suse.de"
+}
+
+// sumaform specific variables
+variable "SCC_USER" {
+  type = string
+}
+
+variable "SCC_PASSWORD" {
+  type = string
+}
+
+variable "GIT_USER" {
+  type = string
+  default = null // Not needed for master, as it is public
+}
+
+variable "GIT_PASSWORD" {
+  type = string
+  default = null // Not needed for master, as it is public
+}
+
+variable "ZVM_ADMIN_TOKEN" {
+  type = string
+}
+
+terraform {
+  required_version = "1.0.10"
+  required_providers {
+    libvirt = {
+      source = "dmacvicar/libvirt"
+      version = "0.6.3"
+    }
+  }
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://suma-06.mgr.suse.de/system"
+}
+
+// WORKAROUND: overdrive3 will be replaced with a new ARM server
+//provider "libvirt" {
+//  alias = "overdrive3"
+//  uri = "qemu+tcp://overdrive3.mgr.suse.de/system"
+//}
+
+module "base_core" {
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  name_prefix = "suma-bv-50-"
+  use_avahi   = false
+  domain      = "mgr.suse.de"
+  images      = [ "sles12sp4o", "sles12sp5o", "sles15sp1o", "sles15sp2o", "sles15sp3o", "sles15sp4o", "sles15sp5o", "slemicro51-ign", "slemicro52-ign", "slemicro53-ign", "slemicro54-ign", "slemicro55-ign", "almalinux9o", "centos7o", "libertylinux9o", "oraclelinux9o", "rocky8o", "rocky9o", "ubuntu2004o", "ubuntu2204o", "debian11o", "debian12o", "opensuse155o" ]
+
+  mirror = "minima-mirror-ci-bv.mgr.suse.de"
+  use_mirror_images = true
+
+  testsuite          = true
+
+  provider_settings = {
+    pool        = "ssd"
+    bridge      = "br0"
+    additional_network = "192.168.50.0/24"
+  }
+}
+
+// WORKAROUND: overdrive3 will be replaced with a new ARM server
+//module "base_arm" {
+//  providers = {
+//    libvirt = libvirt.overdrive3
+//  }
+//
+//  source = "./modules/base"
+//
+//  cc_username = var.SCC_USER
+//  cc_password = var.SCC_PASSWORD
+//  name_prefix = "suma-bv-50-"
+//  use_avahi   = false
+//  domain      = "mgr.suse.de"
+//  images      = [ "opensuse154armo", "opensuse155armo" ]
+//
+//  mirror = "minima-mirror-ci-bv.mgr.suse.de"
+//  use_mirror_images = true
+//
+//  testsuite = true
+//
+//  provider_settings = {
+//    pool        = "ssd"
+//    bridge      = "br1"
+//  }
+//}
+
+module "server_containerized" {
+  source             = "./modules/server_containerized"
+  base_configuration = module.base_core.configuration
+  product_version    = "head"
+  name               = "srv"
+  image              = "slemicro55-ign"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:51"
+    memory             = 40960
+    vcpu               = 10
+    data_pool          = "ssd"
+  }
+  runtime = "podman"
+  container_repository = "registry.suse.de/devel/galaxy/manager/head/containers/suse/manager/5.0/server"
+  helm_chart_url = "oci://registry.suse.de/devel/galaxy/manager/head/charts/suse/manager/5.0/server-helm"
+
+  server_mounted_mirror          = "minima-mirror-ci-bv.mgr.suse.de"
```

Comment:
We are probably missing the a disk for the reposync. The host disk will be quickly full. I don't know how it will be manage at the container level 
`  repository_disk_size           = 2048`

Reply:
I can see on sumaform side that repository_disk_size is not supported. I wonder how we can synchronize all the repositories for BV without this disk.

Reply:
Right. This is as of now only available for the normal `server` module.